src/fontMetrics.js

```diff
@@ -106,7 +106,10 @@ var metrics = {
             return sigma21ScriptScript;
         }
         throw new Error("Unexpected style size: " + style.size);
-    }
+    },
+
+    // \nulldelimiterspace=1.2pt, pg. 344 TeXbook
+    nulldelimiterspace: 1.2 / ptPerEm
```

Comment:
There's lots of other measurements like this that appear in fontMetrics.js, that's why I put it there.  I think you're right about it being always in ```pt```.